docs-src/package.json

```diff
@@ -20,17 +19,20 @@
     "gatsby-plugin-sharp": "^2.2.9",
     "gatsby-source-filesystem": "^2.1.8",
     "gatsby-transformer-sharp": "^2.2.5",
+    "harmonium": "file:..",
```

Comment:
Maybe I'm mis-remembering but wasn't this already in another PR recently?

Reply:
It is. npm re-ordered the deps alphabetically when I installed other packages

Reply:
Gotcha